### ISSUE TITLE
Installation issue: Darshan-runtime GCC 9.3.0 Ubuntu 20.04

### ISSUE THREAD

**Issue by eugeneswalker** [LLNL/spack#16930]
Using `spack@develop` with commit id `70c3b0ba0941` from 3-June-2020.

System:
* Ubuntu 20.04
* GCC 9.3.0
* x86_64 and ppc64le

```
$> spack install darshan-runtime
...
==> Error: Failed to install darshan-runtime due to ChildError: ProcessError: Command exited with status 2:
    'make' '-j16'
1 error found in build log:
     99     ar rcs lib/libdarshan-stubs.a lib/darshan-pnetcdf-stubs.o
     100    ../darshan-runtime/lib/darshan-core.c: In function 'darshan_get_exe_and_mounts':
     101    ../darshan-runtime/lib/darshan-core.c:1150:13: warning: ignoring return value of 'fgets', declared with attribute warn_unused_result [-Wunused-re
            sult]
     102     1150 |             fgets(cmdl,DARSHAN_EXE_LEN,fh);
     103          |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     104    /tmp/ccpXnisF.s: Assembler messages:
  >> 105    /tmp/ccpXnisF.s:8281: Error: symbol `__isoc99_fscanf' is already defined
     106    make: *** [Makefile:143: lib/darshan-stdio.po] Error 1
     107    make: *** Waiting for unfinished jobs....

See build log for details:
  /tmp/root/spack-stage/spack-stage-darshan-runtime-3.1.8-75i6acapt7rptrmgkibmuizjlciagw4a/spack-build-out.txt
```

Full build log: [darshan-runtime.txt](https://github.com/spack/spack/files/4724615/darshan-runtime.txt)

@carns  @shanedsnyder @michaelkuhn @carsonwoods

**Comment by shanedsnyder** [comment]
Whoops, I just realized we haven't updated the spack repo with the new Darshan versions released over the last month or so. Sorry about that! I just submitted a new PR (#16934) that should add those.

The build error you are seeing was fixed in version 3.2.0 (you are building 3.1.8 here). The new PR adds up through 3.2.1, so I'd recommend using that once it has been accepted. If you need to get by in the meantime, `darshan-runtime@develop` should also work fine.

**Comment by eugeneswalker** [comment]
Yes! Version 3.2.1 resolves this problem. Thank you!
